app/helpers/application_helper.rb

```diff
@@ -81,7 +81,7 @@ def multi_conditional_field(f, property, enablers)
 
   def confirm_deleting(record, target_controller = nil)
     target_controller ||= controller_name
-    if ['assignments', 'journals', 'hours'].include? target_controller
+    if ['assignments', 'journals', 'hours', 'assignment_journals'].include? target_controller
       confirm_text = t("delete_#{target_controller.singularize}")
     end
     { method: :delete, data: { confirm: confirm_text || t_confirm_delete(record) } }
```

Comment:
You can also pass a default string to [`I18n.translate`](http://www.rubydoc.info/github/svenfuchs/i18n/master/I18n/Base#translate-instance_method) to avoid listing all the controllers:

```ruby
confirm_text = t("delete_#{target_controller.singularize}", default: t_confirm_delete(record))
{ method: :delete, data: { confirm: confirm_text } }
```